CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Changelog
+# 1.82.4
+* Enhancement - Add retries to DockerClient's manifest pull functionality and add utils/reference package [#4150](https://github.com/aws/amazon-ecs-agent/pull/4150)
+* Enhancement - ContainerRestartPolicy ACS and TCS models, TMDS updates [#4158](https://github.com/aws/amazon-ecs-agent/pull/4158)[#4163](https://github.com/aws/amazon-ecs-agent/pull/4163)
+* Enhancement - Add restart tracker and restart policy to ecs-agent module
```

Comment:
missing PR link

Reply:
PR not available

Reply:
https://github.com/aws/amazon-ecs-agent/pull/4162 ?